### PR TITLE
[skip-ci] Tutorial with Dask on CERN batch service

### PR DESCRIPTION
**Pull request by vepadulano**

### CODE REVIEW

File: tutorials/dataframe/distrdf003_dask_lxbatch.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    # At this point, the jobs may or may not have actually started. Thus, it is
+    # not guaranteed that the application already has the requested resources
+    # available.
+    client = Client(cluster)
+    # It is possible to tell the Dask client to wait until the condor scheduler
+    # has started the requested jobs and launched the Dask workers.
+    # The client will wait until 'n_workers' workers have been launched. In this
+    # example, the client waits for 1 worker before continuing with the
+    # application.
+    client.wait_for_workers(n_workers=1)
```
Comment (eguiraud) [inline tutorials/dataframe/distrdf003_dask_lxbatch.py:77]: should we teach this? if you want to mention it, should the code be commented to clarify you don't usually want to run it?

**Review by eguiraud — APPROVED** [review]
LGTM but I'd like Enric to approve as well before merging.

**Review by etejedor — APPROVED** [review]
LGTM, for the record we can keep the usage of `HTCondorCluster` instead of `dask_lxplus.CernCluster`, since the latter is more specific to CERN and the former is more generic.
To be tested with latest LCG (102).

**Comment by vepadulano** [comment]
All the ingredients needed in the tutorial seem to be finally working well together. This is what I get on lxplus with the latest LCG release (LCG104)
```
$which root
/cvmfs/sft.cern.ch/lcg/views/LCG_104/x86_64-centos7-gcc11-opt/bin/root
$python distrdf004_dask_lxbatch.py 
/cvmfs/sft.cern.ch/lcg/views/LCG_104/x86_64-centos7-gcc11-opt/lib/python3.9/site-packages/distributed/security.py:116: DeprecationWarning: support for OpenSSL 1.0.2k-fips  26 Jan 2017 is deprecated, and will be removed in a future release
  warnings.warn(
Waiting for 2 workers to start.
All workers are ready, took 369.31 seconds.
Starting the computations at 2023-09-20 22:27:35.737711
Dataset has 10000 entries
Column x stats:
	mean: 50.19706628187094
	max: 99.99726188834757
	min: 0.0008953968062996864
Computations ended at 2023-09-20 22:28:02.801630, took 27.06 seconds.
```

**Comment by vepadulano** [comment]
Just tested again on lxplus, works still with the same code unchanged. Periodically we'll need to check that it still works, e.g. at newer versions of `dask_lxplus`. But the tutorial is mostly informative about the fact that one can use the HTCondor pools on lxplus via dask. Given that this tutorial is never run in our CI, I will merge the PR as it is already approved.

**Pull request merged**